MapboxNavigation/RouteMapViewController.swift

```diff
@@ -414,11 +416,6 @@ class RouteMapViewController: UIViewController {
         self.navigationView.endOfRouteView = endOfRoute.view
         self.navigationView.constrainEndOfRoute()
         endOfRoute.didMove(toParent: self)
-
-        endOfRoute.dismissHandler = { [weak self] _, _ in
```

Comment:
Now that we're building programmatically, not from the storyboard, this moved to the initializer.